week-2-practise/level-3/4-slice.js

```diff
@@ -5,7 +5,7 @@
 // - insert the replacement value at the provided index
 
 function replace(arr, index, value) {
-  return; // complete this statement
+  return newArr = arr.slice(0, index).concat(value, arr.slice(index + 1)); // complete this statement
```

Comment:
It works, but here's a couple of improvements you can do:
- the comment about the `newArr` still stand (as in the comment above)
- I think it would be nice to split this into multiple variables so it's clear what's happening here - good for readability! :)